Simplified/Reader2/LCP/LCPLibraryService.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  LCPLibraryService.swift
+//  r2-testapp-swift
+//
+//  Created by Mickaël Menu on 01.02.19.
+//
+//  Copyright 2019 Readium Foundation. All rights reserved.
+//  Use of this source code is governed by a BSD-style license which is detailed
+//  in the LICENSE file present in the project repository where this source code is maintained.
+//
+
+#if LCP
+
+import Foundation
+import UIKit
+import R2Shared
+import ReadiumLCP
+
+
+@objc class LCPLibraryService: NSObject, DRMLibraryService {
+  
+  private var lcpService =  R2MakeLCPService()
```

Comment:
where is R2MakeLCPService defined? I can't seem to find it...

Reply:
R2MakeLCPService is defined in r2-lcp-swift